index.html

```diff
@@ -3117,25 +3122,22 @@ <h4><code>unsubscribeevent</code></h4>
             the Web Thing with:
           </p>
           <ul>
-            <li>Method set to <code>POST</code></li>
-            <li>URL set to the URL of the <code>Event</code> resource</li>
+            <li>Method set to <code>DELETE</code></li>
+            <li>URL set to the URL of the <code>Event</code> resource followed by the <code>subscriptionId</code> of a valid subscription</li>
             <li>
               <code>Accept</code> header set to <code>application/json</code>
             </li>
-            <li>
-              <code>Request Payload</code> contains a <code>JSON</code>
-              object with a valid <code>subscriptionId</code>.
-            </li>
-            <p class="ednote">
-              <!-- TODO: define subscriptionId format - urn? -->
-              TODO: define subscriptionId format - urn?
-            </p>
           </ul>
+
+          <pre class="example" title="Unsubscribe using subscriptionId">
+            DELETE /things/lamp/events/overheated/1234-4544-1211 HTTP/1.1
```

Comment:
Assertion `http-webhook-profile-protocol-binding-subscribeevent-4b` says that the subscription ID should be sent in the payload, but here it is included in the URL. I agree it should be in the URL.